.gitignore

```diff
@@ -8,3 +8,4 @@ liquidctl/_version.py
 .DS_Store
 .coverage
 .vscode/
+/venv/
```

Comment:
Is this needed?

Reply:
No.

Reply:
This should work, with your three commits:

1) `git rebase -i head~3` to interactively rebase last three comits
2) Specify `e` for the first commit to edit it, and `d` for the last one to drop it
3) `git reset head^ -- .gitignore` to reset the file as it was before
4) `git checkout -- .gitignore` to remove it from staging
5) `git commit --amend` to update the commit
6) `git rebase --continue` to finish the rebase
7) `git push -f ...` to force push your changes

Reply:
OK that took me longer than expected but that looks a lot nicer now. Thank you!